### PR TITLE
Draft for email presenting the website

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
I will send the email when we get consensus on website.

File: com.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ - News have been moved to homepage
+
+The improvement of the website is an important milestone towards James 3.0 release.
+
+You can view the result online: http://62.210.100.33
+
+Now we will:
+
+ - Welcome feed-backs for one week (until 21/04/2017 10 am UTC+1)
+ - Then we will call for vote of the final result (from 21/04/2017 to 28/04/2017 UTC+1)
```
Comment (mbaechler) [inline com.md]: no hour ?

File: com.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+   - Landing page of old website have been made more user friendly.
+ - News have been moved to homepage
+
+The improvement of the website is an important milestone towards James 3.0 release.
+
+You can view the result online: http://62.210.100.33
+
+Now we will:
+
+ - Welcome feed-backs for one week (until 21/04/2017 10 am UTC+1)
```
Comment (mbaechler) [inline com.md]: who is UTC+1 ? (:

Comment (chibenwa) [inline com.md]: screenshot from 2017-04-14 07-48-49

France. It happens being the afternoon in Vietnam.

File: com.md (hunk 1 above)
Comment (chibenwa) [inline com.md]: I will be in vacation for the end of the vote. Hence I can not engage my self on the hours.

**Pull request closed without merging**